### PR TITLE
Add missing resource support to AU Core Capability Statements (FHIR-46546)

### PR DESCRIPTION
**Pull request by JayMurdoch**
This PR fixes [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)

Changes:
* AU Core Requester CapabilityStatement: add entry for DocumentReference and RelatedPerson
* AU Core Responder CapabilityStatement: add entry for DocumentReference and RelatedPerson
* Change log

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -180,6 +180,7 @@ This change log documents the significant updates and resolutions implemented fr
   - corrected the Observation combined search parameter 'patient+category+status' from SHALL to SHOULD [FHIR-45390](https://jira.hl7.org/browse/FHIR-45390)
   - corrected narrative of Location identifier parameter requester requirements on providing both the system and code values from SHOULD to MAY [FHIR-46209](https://jira.hl7.org/browse/FHIR-46209)
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
+  - added entries for referenced resources, DocumentReference and RelatedPerson, in line with the Must Support requirements in AU Core profiles [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```
Comment (dt-r) [inline input/pagecontent/changes.md:183]: rephrase to just "added missing entry for DocumentReference and RelatedPerson resources."
this text "in line with ..." is like gravity - it applies to all changes ever.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -188,3 +189,4 @@ This change log documents the significant updates and resolutions implemented fr
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
   - corrected narrative of PractitionerRole practitioner.identifier search parameter from Medicare Provider Number to HPI-I [FHIR-47013](https://jira.hl7.org/browse/FHIR-47013)
   - corrected support for Practitioner _id search parameter to be SHOULD instead of SHALL [FHIR-46777](https://jira.hl7.org/browse/FHIR-46777)
+  - added entries for referenced resources, DocumentReference and RelatedPerson, in line with the Must Support requirements in AU Core profiles [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```
Comment (dt-r) [inline input/pagecontent/changes.md:192]: rephrase to just "added missing entry for DocumentReference and RelatedPerson resources."
this text "in line with ..." is like gravity - it applies to all changes ever.

File: input/resources/au-core-requester.xml [hunk 3]
```diff
@@ -807,6 +841,49 @@
...
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org/fhir/StructureDefinition/DocumentReference">http://hl7.org/fhir/StructureDefinition/DocumentReference</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```
Comment (dt-r) [inline input/resources/au-core-requester.xml:858]: The Profile Conformance is SHALL. Apply this here and in the structure.

File: input/resources/au-core-requester.xml [hunk 4]
```diff
@@ -2850,6 +2927,49 @@
...
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson">http://hl7.org.au/fhir/StructureDefinition/au-relatedperson</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```
Comment (dt-r) [inline input/resources/au-core-requester.xml:2944]: The Profile Conformance is SHALL. Apply this here and in the structure.

File: input/resources/au-core-responder.xml [hunk 5]
```diff
@@ -819,6 +851,49 @@
...
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org/fhir/StructureDefinition/DocumentReference">http://hl7.org/fhir/StructureDefinition/DocumentReference</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```
Comment (dt-r) [inline input/resources/au-core-responder.xml:868]: The Profile Conformance is SHALL. Apply this here and in the structure.

File: input/resources/au-core-responder.xml [hunk 6]
```diff
@@ -2859,6 +2934,49 @@
...
+<div class="panel-body">
+<div class="container">
+<div class="row">
+<div class="col-lg-6">
+<span class="lead">Base System Profile</span>
+<br/>
+<a href="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson">http://hl7.org.au/fhir/StructureDefinition/au-relatedperson</a>
+</div>
+<div class="col-lg-3">
+<span class="lead">Profile Conformance</span>
```
Comment (dt-r) [inline input/resources/au-core-responder.xml:2951]: The Profile Conformance is SHALL. Apply this here and in the structure.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:183]: Amended, thanks

File: input/pagecontent/changes.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:192]: Amended, thanks

File: input/resources/au-core-requester.xml (hunk 4 above)
Comment (JayMurdoch) [reply input/resources/au-core-requester.xml:2944]: Applied profile conformance to narrative, bolded SHALL in documentation for narrative and structure.

File: input/resources/au-core-responder.xml (hunk 5 above)
Comment (JayMurdoch) [reply input/resources/au-core-responder.xml:868]: Applied profile conformance to narrative, bolded SHALL in documentation in for narrative and structure.

File: input/resources/au-core-responder.xml (hunk 6 above)
Comment (JayMurdoch) [reply input/resources/au-core-responder.xml:2951]: Applied profile conformance to narrative, bolded SHALL in documentation in for narrative and structure.

File: input/resources/au-core-requester.xml (hunk 3 above)
Comment (JayMurdoch) [reply input/resources/au-core-requester.xml:858]: Applied profile conformance to narrative, bolded SHALL in documentation in for narrative and structure.

File: input/pagecontent/changes.md [hunk 7]
```diff
@@ -180,6 +180,7 @@ This change log documents the significant updates and resolutions implemented fr
   - corrected the Observation combined search parameter 'patient+category+status' from SHALL to SHOULD [FHIR-45390](https://jira.hl7.org/browse/FHIR-45390)
   - corrected narrative of Location identifier parameter requester requirements on providing both the system and code values from SHOULD to MAY [FHIR-46209](https://jira.hl7.org/browse/FHIR-46209)
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
+  - added missing entry for DocumentReference and RelatedPerson resources [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```
Comment (dt-r) [inline input/pagecontent/changes.md:183]: spelling issues. entry should be plural.

File: input/pagecontent/changes.md [hunk 8]
```diff
@@ -188,3 +189,4 @@ This change log documents the significant updates and resolutions implemented fr
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
   - corrected narrative of PractitionerRole practitioner.identifier search parameter from Medicare Provider Number to HPI-I [FHIR-47013](https://jira.hl7.org/browse/FHIR-47013)
   - corrected support for Practitioner _id search parameter to be SHOULD instead of SHALL [FHIR-46777](https://jira.hl7.org/browse/FHIR-46777)
+  - added missing entry for DocumentReference and RelatedPerson resources [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```
Comment (dt-r) [inline input/pagecontent/changes.md:192]: spelling issues. entry should be plural.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: spelling issues. entry should be plural.

File: input/pagecontent/changes.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: spelling issues. entry should be plural.

**Review by dt-r — CHANGES_REQUESTED** [review]
See continued inline conversation on changes.md

File: input/pagecontent/changes.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md]: Amended, thanks

File: input/pagecontent/changes.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md]: Amended, thanks

**Review by dt-r — APPROVED** [review]

**Pull request merged**